include/claims_tests.h

```diff
@@ -0,0 +1,31 @@
+//  Copyright (c) 2021-22, VMware Inc, and the Certifier Authors.  All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef __CLAIMS_TESTS_H__
+#define __CLAIMS_TESTS_H__
+
+// RESOLVE: Need this? using namespace certifier::framework;
```

Comment:
@yelvmw -- I think these namespaces are not needed by these tests. Things worked just fine w/o these namespaces.

Let me know ... RESOLVE can probably go away.

Reply:
Did the tests already use the namespaces? It's probably OK to just ignore it.

Reply:
Took these comments out... Not specifying namespaces seemed to not make a diff for the builds and tests.